### PR TITLE
chore: Release error field mapping/names

### PR DESCRIPTION
**Pull request by Danwakeem**
## Description
Release to fix the error field mapping/name changes

### CODE REVIEW

File: node/packages/aws-lambda-otel-extension/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,19 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [0.4.2](https://github.com/serverless/runtime/compare/@serverless/aws-lambda-otel-extension@0.4.1...@serverless/aws-lambda-otel-extension@0.4.2) (2022-06-15)
+
+### Bug Fixes
+
+- Updated error field mapping and names ([94929fa](https://github.com/serverless/runtime/commit/94929fae2efc6a39df7ff4266c9747023a797ea0))
+- Updated timeout error message ([f67571e](https://github.com/serverless/runtime/commit/f67571ef5df790af2ae49f7a312eb96b38285cb0))
```
Comment (medikoo) [inline node/packages/aws-lambda-otel-extension/CHANGELOG.md]: I would remove this from changelog, as after all we've removed that message

Comment (Danwakeem) [reply node/packages/aws-lambda-otel-extension/CHANGELOG.md]: Okay will do 👍 Is it okay to manually remove this from the change log or do I need to do anything specific with the commit history?

Comment (medikoo) [reply node/packages/aws-lambda-otel-extension/CHANGELOG.md]: It's ok to do manually. There's no obligation to list all commits, and messaging can also be improved if needed

Comment (Danwakeem) [reply node/packages/aws-lambda-otel-extension/CHANGELOG.md]: Awesome, that is good to know for the future 😎  I updated the change log ✅

**Review by medikoo — APPROVED** [review]
👍

**Pull request merged**